features/ccd/et1_validating_all_fields_in_ccd.feature

```diff
@@ -114,7 +114,7 @@ Feature:
   Scenario: Single claimaint - RTF format
     Given an employee making a claim by uploading a Rich Text Format document
     When the completed form is submitted
-    Then the RTF file should be present in CCD
```

Comment:
I think we still need to prove the file made it to ccd - it's just not an rtf.  We can read the rtf file and we will be able to get something from it such as the title (normally the first line of the text)

If we remove this test and this code stops working, how would we know ?